src/day7.rs

```diff
@@ -0,0 +1,47 @@
+use aoc_utils::DayInfo;
+use aoc_utils::DaySolver;
+use std::cmp::min;
+
+pub struct Day7;
+
+impl DaySolver for Day7 {
+    type Output = i32;
+
+    const INFO: DayInfo = DayInfo::with_day_and_file("day7", "data/day7.txt");
+
+    fn solution(s: &str) -> anyhow::Result<<Self as DaySolver>::Output> {
+        let positions: Vec<i32> = s
+            .split(',')
+            .map(|position| position.parse::<i32>().unwrap())
+            .collect();
+            
+        let maximum_position = positions.iter().max().unwrap();
+
+        let cost = |location: i32| -> i32 {
+            positions
+                .iter()
+                .map(|position| {
+                    let res = location - position;
+                    res.abs()
+                })
```

Comment:
```suggestion
               .map(|position| position.abs_diff(location) as i32)
```

Jeżeli typ elementów wektora `positions`  jak również `type Output` byłby `u32` to rzutowanie nie byłoby potrzebne.